packages/@expo/cli/package.json

```diff
@@ -50,7 +50,6 @@
     "@expo/plist": "^0.1.0",
     "@expo/prebuild-config": "6.7.0",
     "@expo/rudder-sdk-node": "1.1.1",
-    "@expo/server": "^0.3.0",
```

Comment:
I would move this to dev dependency only. Right now, it kind of "inherits" the package types because everything is hoisted in root. But `devDependencies` makes sense, since we still use the types.